README.rst

```diff
@@ -1,5 +1,38 @@
 HCA DSS CLI
 ===========
+This repository contains a prototype for interacting with the replicated data storage system (aka the "blue box") of the Human Cell Atlas.
+
+This prototype uses a local version of the blue box API spec (using Swagger) to generate a command line client that will make requests to the blue box. Currently, the commands listed are: 
+- upload        Upload a file or directory to the cloud, register each of the files, and bundle them together.
+- download      Download a full bundle or file to local.
+- get-search    Returns a list of bundles matching the given simple criteria.
+- post-search   Accepts Elasticsearch JSON query and returns matching bundle identifiers.
+- get-files     Returns a list of files matching given criteria.
+- head-files    Get the associated metadata from a file with a given uuid.
+- put-files     Create a new file with a given UUID. The file content is passed in through a cloud URL. The file on the cloud provider must have metadata set reflecting the file checksums and the file content type.
+- get-bundles   Returns a list of bundles matching given criteria.
+- put-bundles   Create a new bundle with a given UUID. The list of files UUID+versions to be included must be provided.

```

Comment:
Consider making sure these docstrings are present in the data-store swagger YAML, and (for basic wrapper commands, not upload/download) in `hca --help`. Then consider replacing this block of text with something like "Please run `hca --help` to see an overview of available commands".